{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Source \"Freezable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:19:1:\n   |\n19 | import \"Freezable.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"KeyGetters.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:20:1:\n   |\n20 | import \"KeyGetters.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"MainGovernance.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:21:1:\n   |\n21 | import \"MainGovernance.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"ForcedTrades.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:22:1:\n   |\n22 | import \"ForcedTrades.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"ForcedTradeActionState.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:23:1:\n   |\n23 | import \"ForcedTradeActionState.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"ForcedWithdrawals.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:24:1:\n   |\n24 | import \"ForcedWithdrawals.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"ForcedWithdrawalActionState.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:25:1:\n   |\n25 | import \"ForcedWithdrawalActionState.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"SubContractor.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c43f5526124877F9125E3B48101DcA6D7c6B4Ea3_PerpetualForcedActions.sol:26:1:\n   |\n26 | import \"SubContractor.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
